demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -183,6 +200,18 @@ def handler(self) -> XSOAR_Handler:
     def data(self) -> dict:
         return get_file(self.path, keep_order=False)
 
+    @property
+    def original_data(self) -> dict:
+        return get_file(self.path, keep_order=True)
+
+    def save(self):
```

Comment:
there is some code duplication between the `save` methods in Pack and in Content Item, which could lead to bugs in the future. There is even a bug here, since you miss handling the `marketplaces` field.

I suppose implementing a method in the `BaseContentWithPath` which receives the data and the path to write to, and the subclass calls it. For instance:
```
def save(self):
            self._save(self.path, self.original_data)
```
```
def save(self):
      self._save(self.metadata_path, self.original_data)
```
Where we add `metadata_path` and `original_data` to the `Pack` object.